tests/system/action/projector/test_previous.py

```diff
@@ -0,0 +1,88 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorPrevious(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.set_models(
+            {
+                "meeting/1": {},
+                "projector/2": {"meeting_id": 1},
+                "projector/3": {
+                    "current_projection_ids": [1, 2],
+                    "preview_projection_ids": [3, 4],
+                    "history_projection_ids": [6, 7],
+                    "meeting_id": 1,
+                },
+                "projector/4": {
+                    "current_projection_ids": [],
+                    "preview_projection_ids": [],
+                    "history_projection_ids": [5],
+                    "meeting_id": 1,
+                },
+                "projection/1": {
+                    "current_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 100,
+                    "stable": True,
+                },
+                "projection/2": {
+                    "current_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 98,
+                },
+                "projection/3": {
+                    "preview_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 99,
+                },
+                "projection/4": {
+                    "preview_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 100,
+                },
+                "projection/5": {
```

Comment:
can you make this the last projection? A reader can than read the projections for projector 3 in order.

Reply:
Done.